content/zh/docs/reference/access-authn-authz/rbac.md

```diff
@@ -0,0 +1,2137 @@
+translator:
+- liyinda246
+reviewers:
+- tengqm
+title: 使用 RBAC 鉴权
+content_template: templates/concept
+weight: 70
+---
```

Comment:
Please replace line 1-4 with "---". We don't need to sign the translator/reviewer names. The page preview doesn't look good at the moment.

Reply:
ok

Reply:
How to complete the approval? can you approval it?

Reply:
please delete line 1-3, the preview is still incorrect: https://deploy-preview-19276--kubernetes-io-master-staging.netlify.com/zh/docs/reference/access-authn-authz/rbac/

Reply:
I have deleted line 1-3